app/models/event_stream.rb

```diff
@@ -62,13 +62,13 @@ def self.group_and_level(event_type)
 
     group = egroups.detect do |_, value|
       GROUP_LEVELS
-        .detect { |lvl| value[lvl]&.any? { |typ| typ.kind_of?(String) && typ == event_type } }
+        .detect { |lvl| value[lvl]&.any? { |typ| !typ.starts_with?("/") && typ == event_type } }
         .tap { |level_found| level = level_found || level }
     end&.first
 
     group ||= egroups.detect do |_, value|
       GROUP_LEVELS
-        .detect { |lvl| value[lvl]&.any? { |typ| typ.kind_of?(Regexp) && typ.match(event_type) } }
+        .detect { |lvl| value[lvl]&.any? { |typ| typ.starts_with?("/") && typ.match(event_type) } }
```

Comment:
Super minor, but we don't need the match object, so we should use `match?` over `match` to allocate less.

```suggestion
        .detect { |lvl| value[lvl]&.any? { |typ| typ.starts_with?("/") && typ.match?(event_type) } }
```

Reply:
Also, TIL you can pass `.match` a string-that-looks-like-a-regex and it will work!

Reply:
Actually, this can't work...

`typ` is the regex-as-a-string, so while match is a method on string it's expecting the regex-looking-thing as the parameter instead.   Compare:

```ruby
[7] pry(main)> "Some.+Event".match("SomeActualEventName")
=> nil
[8] pry(main)> "SomeActualEventName".match("Some.+Event")
=> #<MatchData "SomeActualEvent">
```

Reply:
I think what you want here is 

```suggestion
        .detect { |lvl| value[lvl]&.any? { |typ| typ.starts_with?("/") && event_type.match?(typ[1..-1]) } }
```